laravel/app/Models/Event.php

```diff
@@ -128,7 +128,7 @@ public function days($hideEmpty = false)
                 if($hideEmpty)
                 {
                     // check to see if the current day is in the list of shift_dates we prepared earlier
-                    if(in_array( $date->format('Y-m-d'), $shift_dates ))
+                    if(!in_array( $date->format('Y-m-d'), $shift_dates ))
```

Comment:
why did you say ! here? when I first opened up your branch I couldn't see the changes in the code, there was actually no visible slots, and a lot of taken slots in my database.

Reply:
I can't reproduce that behavior in my local test instance. When I remove the negation, I only see empty days. 

The original conditional statement checked for ->get()->isEmpty(), so my understanding is that the code in this block should only execute on days without a slots, since it just advances the $date by one day and continues the loop, without adding it to $days.

Can you confirm?

Reply:
@Meleeman01 He said ! because `$shift_dates` contains a list of every date which has shifts on the schedule. So, when hide empty is true and the current date is not in the list of shifts in the schedule, skip to the next day.